cli/js/buffer.ts

```diff
@@ -161,25 +151,6 @@ export class Buffer implements Reader, SyncReader, Writer, SyncWriter {
     this.#reslice(m);
   }
 
-  async readFrom(r: Reader): Promise<number> {
-    let n = 0;
-    while (true) {
-      try {
-        const i = this.#grow(MIN_READ);
-        this.#reslice(i);
-        const fub = new Uint8Array(this.#buf.buffer, i);
-        const nread = await r.read(fub);
```

Comment:
Sorry, I wasn't very specific, readFrom should still be async. Just Buffer should remove its read() and write() methods.